workshops/views.py

```diff
@@ -178,6 +178,16 @@ class SiteUpdate(LoginRequiredMixin, UpdateViewContext):
     slug_url_kwarg = 'site_domain'
     template_name = 'workshops/generic_form.html'
 
+
+@login_required
+def site_delete(request, site_domain):
+    """Delete specific site."""
+    site = get_object_or_404(Site, domain=site_domain)
+    site.delete()
+
+    messages.success(request, 'Site was deleted successfully.')
+    return redirect(reverse('all_sites'))
```

Comment:
Maybe we can set that message in a query parameter here, so we're not racing with any other requests that session may have in flight.